webui/react/src/types.ts

```diff
@@ -446,6 +468,7 @@ export const ExperimentConfig = t.intersection([
   t.partial({
     checkpointStorage: CheckpointStorage,
     description: t.string,
+    integrations: t.union([t.null, t.undefined, PachdIntegration]),
```

Comment:
the union's unnecessary here, no? we're already in a partial so this key can already be undefined.